packages/create-miniapp-cli/template/config/_tailwind.prettier.config.js

```
@@ -0,0 +1,4 @@
+/** @type {import('prettier').Config & import('prettier-plugin-tailwindcss').PluginOptions} */
+export default {
+  plugins: ["prettier-plugin-tailwindcss"],
```

ESLint

[error] Expected indentation of 4 spaces but found 2.

(@typescript-eslint/indent)